Content.Server.Database/Model.cs

```diff
@@ -136,7 +136,7 @@ public class Antag
         [Column("antag_name")] public string AntagName { get; set; } = null!;
     }
 
-    public enum DbPreferenceUnavailableMode
+    public enum DbPreferenceUnavailableMode : byte
```

Comment:
This one also looks to be a DB value.

Like I said it miigghhhttt be fine... but it probably needs a migration.